PythonFiles/Blast.py

```diff
@@ -16,14 +16,14 @@ def runBlast():
     @staticmethod
     def make_blast_db(wgseqs, arguments):
```

Comment:
We should also add a way to specify the target path here. This should be done in such a way that the blastquery function knows where the db's are.